src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.IntegrationAccount.tests.ps1

```diff
@@ -761,5 +761,150 @@ InModuleScope Arcus.Scripting.IntegrationAccount {
                 }
             }
         }
+        Context "Uploading Partners into an Azure Integration Account" {
+            It "Try to upload single partner to unexisting Integration Account fails" {
+                # Arrange
+                $resourceGroupName = $config.Arcus.ResourceGroupName
+                $integrationAccountName = "unexisting-integration-account"
+                $partnerFilePath = "$PSScriptRoot\Files\IntegrationAccount\Partners\partner1.json"
+                $partner = Get-ChildItem($partnerFilePath) -File
+
+                # Act
+                { Set-AzIntegrationAccountPartners -ResourceGroupName $resourceGroupName -Name $integrationAccountName -PartnerFilePath $partner.FullName -ErrorAction Stop} |
```

Comment:
```suggestion
                { Set-AzIntegrationAccountPartners -ResourceGroupName $resourceGroupName -Name $integrationAccountName -PartnerFilePath $partner.FullName} |
```

Can be removed, no?

Reply:
When I remove it, the test fails. other tasks have this as well (for example `Try to upload single schema to unexisting Integration Account fails` and `Try to upload single map to unexisting Integration Account fails`. Probably necessary because this is a test that checks for a failure.

Reply:
Aha aha, ok, let's leave it! 👍